eatimad.html

```diff
@@ -0,0 +1,86 @@
+
+<doctype class="HTML">
+
+
+</doctype>
+
+<html>
+
+<head>
+   
+    <title> eatimad cv </title>
+
+</head>
+   
+<body>
+
+<img src="C:\Users\aamml\OneDrive\سطح المكتب\aaa\first.jpg" alt="try agine" width="300" height="300"></imgy>
```

Comment:
make sure your desktop is on your computer and not the onedrive cause that can cause problems.